index.html

```diff
@@ -9,7 +9,7 @@
   <body>
     <div class="welcome-container">
       <h1 class="welcome-container__headline">Welcome</h1>
-      <p class="welcome-container__paragraph">TODO: General description of how the website works</p>
+      <p class="welcome-container__paragraph"></p>
```

Comment:
The best way to handle TODOs is to remove them :D

Reply:
Lol